dashboard/templates/session.html

```diff
@@ -56,18 +56,12 @@ <h2>Stats</h2>
             <div class="col-md-12 col-sm-6 col-xs-12">
               <ul class="stats-overview session">
                 <li>
-                  <span class="name">Data Transferred Total (KB)</span>
+                  <span class="name">Data Transferred</span>
```

Comment:
It's a bit confusing to see 3 amounts - data sent, data received and data transfered. It's not so obvious for me that data transfered is total of data received and data sent. Maybe it's worth keeping "Total" in the label?